src/p2p/net_node.h

```diff
@@ -172,6 +172,7 @@ namespace nodetool
     virtual void on_connection_close(p2p_connection_context& context);
     virtual void callback(p2p_connection_context& context);
     //----------------- i_p2p_endpoint -------------------------------------------------------------
+    virtual bool relay_notify_to(int command, const std::string& data_buff, const std::list<boost::uuids::uuid> &connections);
```

Comment:
could probably call this "relay_notify_to_multiple" or similar, just to make for easier understanding on a cursory glance.